### ISSUE TITLE
Upload to container/Download from container : select file/path does nothing

### ISSUE THREAD

**Issue by vhaudiquet** [marhkb/pods#703]
Clicking on 'select a path' / 'select a file' on the upload to container or download from container dialog does nothing.

### Reproduction steps

Create a container, open container informations, try to upload/download a file and click on said button.

### Environment

- Pods version: 1.2.1 (compiled by hand, from ArchLinux AUR pods-git package, using rust 1.69)
- Podman version 4.5.1
- OS: ArchLinux (linux-zen 6.3.7-zen1-1-zen), with Gnome 44.2 on Wayland

**Comment by marhkb** [comment]
Hello. I can't reproduce. Could your provide logs?

```
journalctl _COMM=pods -f
```

**Comment by vhaudiquet** [comment]
This does not happen on another PC i have and i'm using right now, that has the exact same configuration. I think it was an edge case of some sort, but it happened multiple times on my other machine. When i get back to that machine i'll try again and look at the logs

**Comment by vhaudiquet** [comment]
![Capture d’écran du 2023-06-20 10-48-10](https://github.com/marhkb/pods/assets/9274077/06f8e6d4-57d2-4acd-8aaa-efd3b7c64065)

The log does not seem to change when i click, and nothing happens still.

Note that this happens on both computers i have running Arch right now.

**Comment by marhkb** [comment]
Is `libportal-gtk4` installed? Does anything else pop in `journalctl -f` when you hit the button?

**Comment by vhaudiquet** [comment]
libportal-gtk4 is indeed installed : 
```
1 extra/libportal-gtk4 0.6-1 (13.0 KiB 37.6 KiB) (Installé)
    GIO-style async APIs for most Flatpak portals - GTK 4 backend
```

The only thing that pops up is  this : 
![Capture d’écran du 2023-06-21 10-44-41](https://github.com/marhkb/pods/assets/9274077/969bd7ef-bf73-4639-80c7-3adfe8a62721)

But it does not seem to be entirely related (it pops up a little bit later).

**Comment by marhkb** [comment]
I can't explain. I tried an Arch distrobox, but couldn't reproduce.

Here are some things to try/consider 

1. Do other Arch users have similar issues?
2. Does the dialog show up when clicking on `Select...` under the image build dialog?
3. Have you tried changing the interface language to English?
4. How does the flatpak package behave? Does the dialog show up here?

**Comment by vhaudiquet** [comment]
Alright, I fixed the issue on my machines.

I needed more packages than `libportal-gtk4`, the issue was fixed after i installed : 
- `xdg-desktop-portal`
- `xdg-desktop-portal-gtk`
- `xdg-desktop-portal-gnome`

I think these packages (or at least the first two i guess) should be added as dependencies to the ArchLinux/AUR packages. I don't think that you are the maintainer of that so i guess i'll comment there too.

Thank you for your time !
